src/shared/components/data-table/dataTableTypes.ts

```diff
@@ -0,0 +1,154 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ReactNode } from 'react';
+
+export enum SortingDirection {
+  ASC = 'ascending',
+  DESC = 'descending',
+  DEFAULT = 'default' // Do we need this?
```

Comment:
I don't think so. Unless what you mean is that the default option is unsorted; in which case it would be more appropriate to call is something like NONE

Reply:
Changed it to `NONE`